packages/block-editor/src/hooks/layout.js

```diff
@@ -265,6 +268,66 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 	);
 }
 
+function ChildLayoutPanel( { setAttributes, attributes } ) {
+	const { style = {} } = attributes;
+	const { layout = {} } = style;
+	const { selfStretch, flexSize } = layout;
+
+	return (
+		<InspectorControls>
+			<PanelBody title={ __( 'Child Layout' ) }>
+				<ToggleGroupControl
+					label={ __( 'Relative size' ) }
+					value={ selfStretch || 'hug' }
+					onChange={ ( value ) => {
+						setAttributes( {
+							style: {
+								layout: {
+									...layout,
+									selfStretch: value,
+								},
+							},
+						} );
+					} }
+					isBlock={ true }
+				>
+					<ToggleGroupControlOption
+						key={ 'hug' }
+						value={ 'hug' }
+						label={ __( 'Hug' ) }
+					/>
+					<ToggleGroupControlOption
+						key={ 'fill' }
+						value={ 'fill' }
+						label={ __( 'Fill' ) }
+					/>
+					<ToggleGroupControlOption
+						key={ 'fixed' }
+						value={ 'fixed' }
```

Comment:
Could "fixed" cause confusion with the fixed/sticky position support once we have both of these features in? I wonder if there's another name we can use for entering a custom value 🤔

Reply:
That's a good question! I'd defer to design on that one 😅